portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/ProfileMgtClientServiceImpl.java

```diff
@@ -147,16 +147,15 @@ public ProfileEntry getProfile(String profileName) throws UserPortalUIException
 
         List<MetaClaim> metaClaims = profileEntry.getClaims().stream()
                 .map(claimConfigEntry -> new MetaClaim(IdentityMgtConstants.CLAIM_ROOT_DIALECT,
-                        claimConfigEntry.getClaimURI()))
-                .collect(Collectors.toList());
+                        claimConfigEntry.getClaimURI())).collect(Collectors.toList());
 
         List<Claim> claims;
         try {
             claims = getRealmService().getIdentityStore().getClaimsOfUser(uniqueUserId, metaClaims);
         } catch (IdentityStoreException e) {
             LOGGER.error(String.format("Failed to get the user claims for user - %s", uniqueUserId), e);
-            throw new UserPortalUIException(String.format("Failed to get the user claims for the profile - %s",
-                    profileName));
+            throw new UserPortalUIException(
+                    String.format("Failed to get the user claims for the profile - %s", profileName));
```

Comment:
@GayanM Failed to get the user claims for the profile - %s, for user %s", profileName, uniqueUserId)